_download-meta/openttd-nightlies.md

```diff
@@ -2,9 +2,14 @@
 folder_old_infrastructure: nightlies/trunk
 ---
 
-For OpenTTD you can use the original Transport Tycoon Deluxe data files (you need to own a Transport Tycoon Deluxe CD).
-There are also the free alternatives: [OpenGFX (graphics)](http://dev.openttdcoop.org/projects/opengfx), [OpenSFX (sound)](http://dev.openttdcoop.org/projects/opensfx), and [OpenMSX (music)](http://dev.openttdcoop.org/projects/openmsx).
-These can be installed automatically by the Windows and OS/2 installers.
-Please refer to the [readme](https://proxy.binaries.openttd.org/openttd-nightlies/@@version@@/README.md) for more information.
 
-You can download the free alternatives here: [download OpenGFX](../opengfx-releases/latest.html), [download OpenSFX](../opensfx-releases/latest.html), and [download OpenMSX](../openmsx-releases/latest.html).
```

Comment:
Why you no longer tell people where to download these? Only Windows has an installer; possibly it is good to tell people how to download these files exactly in case they don't use the installer? (and the dev.openttdcoop.org  links are not really clear; even I get lost in finding which files to download :D)